models.py

```diff
@@ -332,3 +336,64 @@ def get_peaks_for_display():
                     peaks[name] = peaks[name].format('MMM D, YYYY')
 
         return peaks
+
+    def __repr__(self):
+        return 'Variable:' + self.id
+
+
+class RwrAccountType(Enum):
+    INVASION = 'INVASION'
+    PACIFIC = 'PACIFIC'
+
+
+class RwrAccount(db.Model):
+    __tablename__ = 'rwr_accounts'
+    __table_args__ = (db.Index('type_username_idx', 'type', 'username'), )
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+
+    type = db.Column(db.Enum(RwrAccountType), nullable=False)
+    username = db.Column(db.String(16), nullable=False)
+    created_at = db.Column(ArrowType, default=arrow.utcnow().floor('minute'), nullable=False)
+    updated_at = db.Column(ArrowType, default=arrow.utcnow().floor('minute'), nullable=False)
+
```

Comment:
We need one more boolean column to know if this RWR account has any stats or not to prevent making too much SQL queries.